doc/nrf/config_and_build/configuring_app/cmake/index.rst

```diff
@@ -85,10 +85,9 @@ The following table lists the most common ones used in the |NCS|:
        | This variable is deprecated and is being gradually replaced by :makevar:`FILE_SUFFIX`, but :ref:`still required for some applications <modifying_build_types>`.
      - ``-DCONF_FILE=prj_<build_type_name>.conf``
    * - ``-S`` (west) or :makevar:`SNIPPET` (CMake)
-     - | Select one of the :ref:`zephyr:snippets` to add to the application firmware during the build.
-       | The west argument ``-S`` is more commonly used.
-     - | ``-S <name_of_snippet>``
-       | ``-DSNIPPET=<name_of_snippet>`` (``-Dimage-name_SNIPPET`` for images)
+     - Select one of the :ref:`zephyr:snippets` to add to the application firmware during the build.
+     - | ``-S <name_of_snippet>`` (applies the snippet to all images)
+       | ``-DSNIPPET=<name_of_snippet>`` (``-Dimage-name_SNIPPET=<name_of_snippet>`` for images)
```

Comment:
There are at least three version for the illustrative image name:
`-Dimage-name_SNIPPET`
`-D<your_image>_SNIPPET`
`-D<your_app_name>_SNIPPET`

Maybe we could select just one of these.

Reply:
Aligned the naming now, thanks!